packages/core/src/rum/__tests__/DdRum.test.ts

```diff
@@ -384,6 +362,13 @@ describe('DdRum', () => {
         });
     });
 
+    describe('DdRum.stopSession', () => {
+        it('calls the native API', async () => {
+            await DdRum.stopSession();
+            expect(NativeModules.DdRum.stopSession).toHaveBeenCalledWith();
```

Comment:
shouldn't it just be `toHaveBeenCalled` here?

```suggestion
            expect(NativeModules.DdRum.stopSession).toHaveBeenCalled();
```

Reply:
With `toHaveBeenCalledWith()` I'm also checking that no argument was passed to the method.

Reply:
ok, makes sense. non-obvious API usage